pkg/spec3/path.go

```diff
@@ -35,15 +35,41 @@ type Paths struct {
 
 // MarshalJSON is a custom marshal function that knows how to encode Paths as JSON
 func (p *Paths) MarshalJSON() ([]byte, error) {
```

Comment:
This function was updated because we did not filter for the "/" prefix before.

Reply:
you could have split that in a separate PR/commit maybe? Maybe if we want to backport? I don't think it matters.

Reply:
why was it a problem for paths from the input object to be serialized

Reply:
It wasn't a problem, but would bring inconsistency between jsonv2 and jsonv1 marshalling